javascripts/background_scripts/posting_process.js

```diff
@@ -82,6 +82,7 @@ var postingProcess = {
     } else {
       var notification = new Notification("There is already a pending post",
         {icon: "images/logo_48.png"});
+      notification.show();
```

Comment:
I didn't get your last point - 'adding the right JSHint comments to declare globals' part. Which issues are you talking about here, of which file?